libs/sdk-ui-dashboard/src/presentation/dashboardList/DashboardList.tsx

```diff
@@ -0,0 +1,98 @@
+// (C) 2020-2022 GoodData Corporation
+import React from "react";
+import { useIntl } from "react-intl";
+import { Dropdown, DropdownList } from "@gooddata/sdk-ui-kit";
+
+import { DashboardListItem } from "./DashboardListItem";
+import { DashboardListButton } from "./DashboardListButton";
+import { dashboardMatch } from "../drill/utils/dashboardPredicate";
+import { IDashboardListProps } from "./types";
+
+const alignPoints = [
+    {
+        align: "bl tl",
+    },
+    {
+        align: "tl bl",
+    },
+];
+
+const ITEM_HEIGHT = 25;
+const DROPDOWN_BODY_WIDTH = 240;
+const DROPDOWN_BODY_HEIGHT = 319;
+
+export const DashboardList: React.FunctionComponent<IDashboardListProps> = ({
+    selected,
+    dashboards,
+    onSelect,
+}) => {
+    const selectedDashboard = selected && dashboards.find((d) => dashboardMatch(d.id, d.ref, selected));
+    const intl = useIntl();
+    const dropdownLabel = selectedDashboard
+        ? selectedDashboard.title
+        : intl.formatMessage({
+              id: "configurationPanel.drillConfig.selectDashboard",
+          });
+    const [searchString, setSearchString] = React.useState("");
```

Comment:
Convention in our codebase are named imports  - `import { useState } from "react"`